functions/src/modules/email.functions.ts

```diff
@@ -0,0 +1,65 @@
+import * as functions from 'firebase-functions';
+import * as SibApiV3Sdk from 'sib-api-v3-sdk';
+const sendInBlueSecretKey = functions.config().send_in_blue.key;
+const defaultClient = SibApiV3Sdk.ApiClient.instance;
+let apiKey = defaultClient.authentications['api-key'];
+apiKey.apiKey = sendInBlueSecretKey;
+
+const apiInstance = new SibApiV3Sdk.SMTPApi();
+
+export const sendEmail = async (to, bcc, templateId, params) => {
+    let sendEmail = new SibApiV3Sdk.SendSmtpEmail();
+
+    sendEmail = {
+        to: [{ email: to }],
+        bcc,
+        templateId,
+        params
+    };
```

Comment:
It is a bad practice to name a variable as a function. And the variable should be a noun, not a verb. And you're overwriting it here. So three issues here.